dips/dip-10.md

```diff
@@ -101,130 +110,87 @@ struct DiemIdDomainEvent {
 
-**Request:**
-```json
-{
-   "sender": "<User Object>",
-   "receiver_id": "<Receiver Diem ID>"
-}
 ```
-
-| Field 	   | Type 	     | Required? 	| Description 	           |
-|-------	   |------	     |-----------	|-------------	           |
-|sender        |[User Object](https://github.com/diem/dip/blob/master/dips/dip-10.md#user-object)  | Y            | Information about the sender |
-|receiver_id   |str          | Y            | Diem ID of the receiver |
-
-**Response:**
-```json
 {
-   "receiver": "<User Object>"
+   "_ObjectType": "CommandRequestObject",
+    "command_type": "ReferenceIDCommand",
+    "command": {
+	    "_ObjectType": "ReferenceIDCommand",
+	    "sender": "alice@avasp",
+	    "sender_address": "f72589b71ff4f8d139674a3f7369c69b",
+	    "recipient": "bob@bvasp",
+	    "reference_id": "5b8403c9-86f5-3fe0-7230-1fe950d030cb", 
+    },
+    "cid": "12ce83f6-6d18-0d6e-08b6-c00fdbbf085a",
 }
 ```
 
-| Field 	   | Type 	     | Required? 	| Description 	             |
-|-------	   |------	     |-----------	|-------------	             |
-|receiver      |[User Object](https://github.com/diem/dip/blob/master/dips/dip-10.md#user-object)  | Y            | Information about the receiver |
+**CommandRequestObject:**
 
-* Receiver Diem ID domain must belong to the Receiver VASP. Otherwise, Receiver VASP must return the `400` error code.
-* Sender Diem ID must have a Diem ID domain that belongs to Sender VASP. If this is not the case, the Receiver VASP should return the `400` error code 
-* Receiver VASP may optionally choose how much information to share based on the sender Diem ID. If Receiver VASP decides to not share data, it should return an empty `User Object` with only `diem_id` field set
-* If user is not found, the VASP should return error code `404`  
+| Field 	    | Type 	     | Required? 	| Description 	           |
+|-------	    |------	     |-----------	|-------------	           |
+| _ObjectType   | str        | Y | Fixed value: `CommandRequestObject`|
+| command_type  | str        | Y | Fixed value: `ReferenceIDCommand`|
+| command       | Command object | Y | The Command to sequence. |
+| cid           | str         | Y            | A unique identifier for the Command. Should be a UUID according to [RFC4122](https://tools.ietf.org/html/rfc4122) with "-"'s included. |
 
-## Pay endpoint
+**CommandObject:**
 
-**TODO: This endpoint is going to be merged into DIP 1**
+| Field 	    | Type 	     | Required? 	| Description 	           |
+|-------	    |------	     |-----------	|-------------	           |
+| _ObjectType   | str    | Y | Fixed value: `ReferenceIDCommand`|
+| sender        | str          | Y            | Sender's full Diem ID |
+| sender_address| str          | Y            | Sender's on-chain address |
+| recipient     | str          | Y            | Receiver's full Diem ID |
+| reference_id  | str          | Y            | Reference ID of this transaction to be included into payment metadata |
```

Comment:
should we state that this reference id is also uuid formated? do we say this somewhere in this doc?